Suko/View Controller/SUKLoginViewController.m

```diff
@@ -36,15 +36,17 @@ - (IBAction)tapLoginButton:(id)sender {
     
     [self checkEmptyField];
 
+    __weak __typeof(self) weakSelf = self;
     [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
         if (error != nil) {
             NSLog(@"User log in failed: %@", error.localizedDescription);
```

Comment:
It seems like nothing is displayed to the user if they fail to login. Are you planning on displaying a failure message?